tfx/tools/cli/handler/dag_runner_patcher.py

```diff
@@ -56,11 +56,13 @@ def __init__(self, call_real_run=True):
     self._run_called = False
     self._call_real_run = call_real_run
 
+  @abc.abstractmethod
   def _before_run(self, runner: tfx_runner.TfxRunner,
                   pipeline: Union[pipeline_pb2.Pipeline, tfx_pipeline.Pipeline],
                   context: MutableMapping[str, Any]) -> None:
     pass
 
+  @abc.abstractmethod
```

Comment:
Same here (and above `_before_run`). I believe all we need is `@abc.abstractmethod` decorators.

Reply:
ditto